lib/routes/telegram/tglib/channel.ts

```diff
@@ -160,6 +82,4 @@ export default async (ctx) => {
         allowEmpty: ctx.req.param('id') === 'allow_empty',
         description: `@${channelInfo.username} on Telegram`,
     });
-};
-
-module.exports.getMedia = getMedia;
+}
```

Comment:
https://github.com/DIYgod/RSSHub/blob/94a78a70bf72463460c3ff17d7b2872f9236981f/lib/routes/telegram/tglib/channel.ts#L65

Do not slice `title`, RSSHub will handle it.
https://github.com/DIYgod/RSSHub/blob/7a8335ee1e12187e483b5685756cce3fb37d42bf/lib/middleware/template.ts#L41-L47